test/extended/setup.sh

```diff
@@ -345,6 +345,10 @@ readonly EXCLUDED_TESTS=(
 
 	# this tests the _kube_ downgrade. we don't support that.
 	"\[Feature:Downgrade\]"
+
+	# upstream flakes:
```

Comment:
We have seen them popping up in the tests pretty regularly, but random. 

I am also fine with keeping them failing tests, but this causes pain for those who want to get their PRs in in the next days. These tests are different, because the plan is to work on them immediately as a follow-up of the rebase merge. Then we revert this commit again, hopefully within some days, the earlier the better.